pkg/encryption/elgamal/encryption.go

```diff
@@ -23,10 +23,10 @@ func New(key *ecdsa.PrivateKey, pub *ecdsa.PublicKey, salt []byte, padding int,
 	return encryption.New(sk, padding, 0, hashfunc), nil
 }
 
-// NewEncrypter constructs an El-Gamal encryptor
+// NewEncryptor constructs an El-Gamal encryptor
 // this involves generating an ephemeral key pair the public part of which is returned
 // as it is needed for the counterparty to decrypt
-func NewEncrypter(pub *ecdsa.PublicKey, salt []byte, padding int, hashfunc func() hash.Hash) (encryption.Encrypter, *ecdsa.PublicKey, error) {
+func NewEncryptor(pub *ecdsa.PublicKey, salt []byte, padding int, hashfunc func() hash.Hash) (encryption.Encrypter, *ecdsa.PublicKey, error) {
```

Comment:
why renaming to `Encryptor`? can we please have `Encrypter`?

Reply:
@zelig had some comment about this.

Reply:
maybe a better name here would be `NewElGamal`. I guess the rename came due to some ambiguity with the interface name but it would be still more informative as `ElGamal` and not a general `Encrypter/or`

Reply:
We merged tjhe rename in #728 why revert it?  objections